website/docs/docs/build/unit-tests.md

```diff
@@ -0,0 +1,253 @@
+---
+title: "Unit tests"
+sidebar_label: "Unit tests"
+description: "Read this tutorial to learn how to use unit tests on your SQL models."
+search_weight: "heavy"
+id: "unit-tests"
+keywords:
+  - unit test, unit tests, unit testing, dag
+---
+:::note closed beta
+
+Unit testing is currently in closed beta for dbt Cloud accounts that have updated to a [versionless environment](/docs/dbt-versions/upgrade-core-in-cloud).
+
+It is available now as an alpha feature for dbt Core v1.8 users.
+
+:::
+
+Historically, the test coverage capabilities of dbt were limited to [“data” tests](/docs/build/data-tests) &dash; testing the quality of input data or the shape of the resulting datasets &mdash; that could only be executed _after_ a model had been built. 
+
+Now, we are introducing a new type of test to dbt - unit tests. In software programming, unit tests validate small portions of your functional code, and they work much the same way here. Unit tests allow you to validate your SQL modeling logic on a small set of static inputs _before_ you materialize your full model in production. Unit tests enable test-driven development, benefiting developer efficiency and code reliability. 
+
+:::note Before you begin
+
+- We currently only support unit testing SQL models.
+- We currently only support adding unit tests to models in your _current_ project.
+- If your model has multiple versions, by default the unit test will run on *all* versions of your model. Read [unit testing versioned models for more information](#unit-testing-versioned-models).
+
+Read the [reference doc](/reference/resource-properties/unit-tests) for more details about formatting your unit tests.
+:::
+
```

Comment:
This feels weird floating here. Let's add a heading
```suggestion

## Example model with unit test

```

Reply:
How about something like this
```suggestion

## Unit testing a model

```